src/Bicep.LangServer/Handlers/BicepBuildCommandHandler.cs

```diff
@@ -82,7 +82,9 @@ private string GenerateCompiledFileAndReturnBuildOutputMessage(string bicepFileP
 
             using (FileStream fileStream = new FileStream(compiledFilePath, FileMode.Create, FileAccess.ReadWrite))
             {
-                TemplateEmitter emitter = new TemplateEmitter(semanticModel, ThisAssembly.AssemblyFileVersion);
+                var emitterSettings = new EmitterSettings(ThisAssembly.AssemblyFileVersion, enableSymbolicNames: false);
```

Comment:
If the environment variable is set, this command should also use symbolic names.

Reply:
Good point, fixed.